src/config/mod.rs

```diff
@@ -3236,7 +3237,7 @@ impl TikvConfig {
         self.quota.validate()?;
         self.causal_ts.validate()?;
 
-        if self.storage.flow_control.enable {
+        if self.storage.flow_control.enable || self.storage.engine == EngineType::RaftKv2 {
```

Comment:
I don't get it. What does "fully verified" mean?

Reply:
today I sometimes found out enabling tablet flow control's performance is slower, not exactly sure yet. Just disable it for now as it's mostly useless in multi-rocksdb when front-end flow control is on. But when disabling it, it will enable rocksdb's write stall, which may have negative impact as well.

Reply:
How about adjusting the configuration in `optmize_for` function? I'm afraid spreading adjustment everywhere is not maintainable.

Reply:
done.